registry.go

```diff
@@ -80,3 +83,20 @@ func (svc *RegistryServiceOp) Delete(ctx context.Context) (*Response, error) {
 	}
 	return resp, nil
 }
+
+// DockerCredentials retrieves a Docker config file with the registry's credentials.
+func (svc *RegistryServiceOp) DockerCredentials(ctx context.Context) ([]byte, *Response, error) {
+	path := fmt.Sprintf("%s/%s", registryPath, "docker-credentials")
+
+	req, err := svc.client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var buf bytes.Buffer
+	resp, err := svc.client.Do(ctx, req, &buf)
+	if err != nil {
+		return nil, resp, err
+	}
+	return buf.Bytes(), resp, nil
```

Comment:
A similar method we have might be the DOKS `GetKubeconfig` that returns the YAML config as `[]byte`, but we nest it inside a response type for responding. Should we follow a similar pattern here?

https://github.com/digitalocean/godo/blob/master/kubernetes.go#L482-L498

Reply:
Thanks. I pushed a commit with this change